Fix compiler warning "permanently hidden by definition of trait"

# Description
The compiler warns of the following
```
[WARNING] /hyperdrive/shared/src/test/scala/za/co/absa/hyperdrive/shared/utils/TestFileUtils.scala:22: warning: imported `SparkTestBase' is permanently hidden by definition of trait SparkTestBase in package utils
[WARNING] import za.co.absa.hyperdrive.shared.utils.SparkTestBase

```

# Solution proposal
Probably it is enough to remove the import, since `SparkTestBase` is defined in the same package as `TestFileUtils` and therefore doesn't need to be explicitly imported.

--- a/shared/src/test/scala/za/co/absa/hyperdrive/shared/utils/TestFileUtils.scala
+++ b/shared/src/test/scala/za/co/absa/hyperdrive/shared/utils/TestFileUtils.scala
@@ -19,7 +19,6 @@ import java.util.UUID
 import org.apache.hadoop.fs.{FileSystem, Path}
 import org.scalatest.{BeforeAndAfter, FlatSpec, Matchers}
 import za.co.absa.commons.io.TempDirectory
-import za.co.absa.hyperdrive.shared.utils.SparkTestBase
 
 class TestFileUtils extends FlatSpec with Matchers with SparkTestBase with BeforeAndAfter {
 

